scipy/stats/_multivariate.py

```diff
@@ -2642,3 +2644,104 @@ def entropy(self):
     method_frozen.__doc__ = doccer.docformat(
         method.__doc__, wishart_docdict_noparams)
     method.__doc__ = doccer.docformat(method.__doc__, wishart_docdict_params)
+
+class rotation_gen(multi_rv_generic):
```

Comment:
How about special_ortho? And ortho for O(N).
Sent from my BlackBerry device on the Rogers Wireless Network

-----Original Message-----
From: Ralf Gommers <notifications@github.com>
Date: Wed, 23 Dec 2015 23:21:15 
To: scipy/scipy<scipy@noreply.github.com>
Reply-To: scipy/scipy <reply@reply.github.com>
Cc: philipdeboer<philip.deboer@gmail.com>
Subject: Re: [scipy] Add SO(N) rotation matrix generator (#5622)

> @@ -2642,3 +2644,104 @@ def entropy(self):
>      method_frozen.__doc__ = doccer.docformat(
>          method.__doc__, wishart_docdict_noparams)
>      method.__doc__ = doccer.docformat(method.__doc__, wishart_docdict_params)
> +
> +class rotation_gen(multi_rv_generic):

``rotation`` is a much too general name, can you propose something more specific?

---
Reply to this email directly or view it on GitHub:
https://github.com/scipy/scipy/pull/5622/files#r48399833